CONTRIBUTING.md

```diff
@@ -1,20 +1,191 @@
+
+For windows installation we will use [choclatey](https://chocolatey.org) a free and open-source package management system for Windows.
+
+- Open the `PowerShell` terminal as an administrator and install the terraform package with 
+
+    ```bash
+    choco install terraform
+    ```
+- To confirm its installation close the current terminal and open a new `PowerShell` terminal as a normal user and use `terraform -help`.
+
+### **Linux installation**
+
+To install Terraform on your Linux distribution, visit the [HashiCorp Tutorial](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) and follow the provided instructions for your specific distribution. After installation, verify Terraform by running `terraform -help` in your terminal.
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+## **Installing Terraform docs**
+
+To install Terraform docs we will be using Docker, terraform-docs can be run as a container by mounting a directory with `.tf` files in it and run the following command:
+
+    ```bash
+    docker run --rm --volume "$(pwd):/terraform-docs" -u $(id -u) quay.io/terraform-docs/terraform-docs:0.17.0 markdown /terraform-docs
+    ```
+If `output.file` is not enabled for this module, generated output can be redirected back to a file:
+
+    ```bash
+    docker run --rm --volume "$(pwd):/terraform-docs" -u $(id -u) quay.io/terraform-docs/terraform-docs:0.17.0 markdown /terraform-docs > doc.md
+    ```
+**Note:** Docker tag `latest` refers to latest stable released version and `edge` refers to HEAD of `master` at any given point in time. And any named version tags are identical to the official GitHub releases without leading `v`.
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+## **Fork the repository**
+
+You can fork the hackforla/ops repository by clicking <a href="https://github.com/hackforla/ops/fork"> <button> <img src="https://user-images.githubusercontent.com/17777237/54873012-40fa5b00-4dd6-11e9-98e0-cc436426c720.png" width="8px"> Fork</button></a>
+. A fork is a copy of the repository that will be placed on your GitHub account.
+
+<details>
+  <summary><strong>Click here</strong> to see a visual example of the `fork` button within the website repo</summary>
+  <p><strong>First, you find the `fork` button on the top right hand side of the [Hack for LA website repo](https://github.com/hackforla/website)</strong></h4>
+  <img src="https://user-images.githubusercontent.com/21162229/137044762-c80f18e8-b785-48fb-a286-81c1117c0178.jpg" />
+  <p><strong>Next, a modal may open and request where you want to fork this website repo. Please click on your avatar or your GitHub username</strong></p>
+  <img src="https://user-images.githubusercontent.com/21162229/137045294-3d46b28c-edbb-410c-98f1-13940ecc5c5f.jpg" />
+</details><br>
+
+**Note:** It should create a URL that looks like the following -> `https://github.com/<your_GitHub_user_name>/website`.
```

Comment:
```suggestion
**Note:** It should create a URL that looks like the following -> `https://github.com/<your_GitHub_user_name>/devops-security`.
```

Reply:
Correction on the line 
```
<summary><strong>Click here</strong> to see a visual example of the `fork` button within the website repo</summary>
  <p><strong>First, you find the `fork` button on the top right hand side of the [Hack for LA website repo](https://github.com/hackforla/devops-security)</strong></h4>

  ```

should be this instead I guess

```
<summary><strong>Click here</strong> to see a visual example of the `fork` button within the website repo</summary>
  <p><strong>First, you find the `fork` button on the top right hand side of the [Hack for LA devops-security repo](https://github.com/hackforla/devops-security)</strong></h4>

  ```

Reply:
Just to confirm from your and for making correction

Reply:
yes, you're right